### PR TITLE
Tweak: PR #1115 code cleanup

### PR DESCRIPTION
**Pull request by 9921622**
code cleanup from pr #1115

### CODE REVIEW

File: src/tags/tests/test_models.py [hunk 1]
```diff
@@ -20,12 +20,17 @@ def setUp(self):
...
 
     def create_quest_and_submissions(self, xp, quest_submission_quantity=1):
-        """ 
-            Creates and returns quest with linked quest submission objects
-            Args
-                xp - amount of xp quest obj will have
-                quest_submission_quantity - how many submissions are crated
-        """ 
+        """
+        Creates and returns quest with linked quest submission objects
```
Comment (tylerecouture) [inline src/tags/tests/test_models.py:24]: ...for self.user

File: src/tags/models.py [hunk 2]
```diff
@@ -8,16 +8,22 @@
...
-    """ 
-        Returns user's total xp over active_semester filtered by tags
-        Args:
-            tags - a queryset or list of Tag objects
-    """ 
+    """
+    Returns user's total xp over active_semester filtered by tags
+
+    Args:
+        tags (list[str], Queryset<Tag>): a queryset or list of Tag objects
```
Comment (tylerecouture) [inline src/tags/models.py:15]: Is the ability to accept a Queryset actually tested?  if not, can you add one?

**Review by tylerecouture — CHANGES_REQUESTED** [review]
Can you add a test to ensure that your method will work with a queryset?  If it doesn't, you can remove the info about querysets in your docs and add it as a new issue (we can worry about it later, not important for our current deadline)

File: src/tags/models.py (hunk 2 above)
Comment (9921622) [reply src/tags/models.py:15]: added test for qs and list

**Review by tylerecouture — APPROVED** [review]
Nice

**Pull request merged**